src/slashcommands/ISlashCommand.ts

```diff
@@ -0,0 +1,6 @@
+// just guessing
+export interface ISlashCommand {
```

Comment:
Only thing I think that needs added is the `description` until the suggestions on parameters gets implemented inside of Rocket.Chat.